certified-connectors/Encodian/Readme.md

```diff
@@ -0,0 +1,149 @@
+
+## Encodian for PowerAutomate
+[Encodian for PowerAutomate](https://www.encodian.com/products/encodian-for-microsoft-power-automate/) - Supercharge your Microsoft Power Automate and Azure Logic Apps solutions with our enhanced document management actions, providing PDF conversion, document merging, watermarks, OCR and advanced document and image clean-up capabilities..
```

Comment:
Space needed here as well. Thanks!

Reply:
All done, pushed to the master branch on  Encodian/PowerPlatformConnectors

Reply:
Thanks!